pkg/actors/actors.go

```diff
@@ -1101,15 +1121,15 @@ func (a *actorsRuntime) Close() error {
 	defer a.wg.Wait()
 
 	if a.closed.CompareAndSwap(false, true) {
-		defer close(a.closeCh)
-		errs := []error{}
+		defer func() { close(a.closeCh) }()
+		if a.checker != nil {
+			a.checker.Close()
+		}
 		if a.placement != nil {
-			err := a.placement.Close()
-			if err != nil {
-				errs = append(errs, fmt.Errorf("failed to close placement service: %w", err))
+			if err := a.placement.Close(); err != nil {
+				return fmt.Errorf("failed to close placement service: %w", err)
 			}
 		}
-		return errors.Join(errs...)
 	}
```

Comment:
Please don't remove the errors.Join, there can be other things to close

```suggestion
	if a.closed.CompareAndSwap(false, true) {
		defer close(a.closeCh)
		errs := []error{}
        if a.checker != nil {
			a.checker.Close()
		}
		if a.placement != nil {
			err := a.placement.Close()
			if err != nil {
				errs = append(errs, fmt.Errorf("failed to close placement service: %w", err))
			}
		}
		return errors.Join(errs...)
	}
```

Reply:
Sure thing, thanks @ItalyPaleAle. Done